orchagent/portsorch.cpp

```diff
@@ -682,24 +839,58 @@ void PortsOrch::doVlanTask(Consumer &consumer)
 
         if (op == SET_COMMAND)
         {
-            /* Duplicate entry */
-            if (m_portList.find(vlan_alias) != m_portList.end())
+            if (m_portList.find(vlan_alias) == m_portList.end())
             {
-                it = consumer.m_toSync.erase(it);
-                continue;
+                if (addVlan(vlan_alias) == false)
+                {
+                    it++;
+                    continue;
+                }
             }
 
-            if (addVlan(vlan_alias))
-                it = consumer.m_toSync.erase(it);
-            else
-                it++;
+            string autostate;
```

Comment:
just wondering if autostate is a `boolean` or a `string`?

Reply:
It is a value from redis, in type of string.  VLAN autostate feature is to be implemented.